### PR TITLE
add new field on project-info page.

### PR DESCRIPTION
**Pull request by ywrsusan**
*Issue number of the reported bug or feature request: #<2441>*

### CODE REVIEW

File: pybossa/cache/helpers.py [hunk 1]
```diff
@@ -29,6 +29,17 @@
 
 session = db.slave_session
 
+def n_gold_tasks(project_id):
+    """Returh the number of gold tasks for a given project"""
```
Comment (dchhabda) [inline pybossa/cache/helpers.py]: typo here

File: pybossa/cache/helpers.py [hunk 2]
```diff
@@ -29,6 +29,17 @@
...
+    query = text('''SELECT COUNT(*) AS n_gold_tasks FROM task
+                WHERE project_id=:project_id
+                AND state !='enrich'
+                AND calibration = 1;''')
+    result = session.execute(query, dict(project_id=project_id))
+    n_gold_tasks = 0
```
Comment (dchhabda) [inline pybossa/cache/helpers.py]: this could be total_gold_tasks to avoid name conflict w/ func name

File: pybossa/cache/projects.py [hunk 3]
```diff
@@ -263,8 +263,16 @@ def n_remaining_task_runs(project_id):
 
 
 def n_expected_task_runs(project_id):
-    """Return total number of expected task_runs of a project."""
-    return n_task_runs(project_id) + n_remaining_task_runs(project_id)
+    """Return total number of expected task_runs of a project (exclude gold task)."""
+    sql = text('''SELECT SUM(n_answers) AS n_task_runs FROM task
+                        WHERE project_id=:project_id
```
Comment (dchhabda) [inline pybossa/cache/projects.py]: indent code here?

File: pybossa/cache/helpers.py (hunk 2 above)
Comment (ywrsusan) [reply pybossa/cache/helpers.py]: thanks for pointing out, will fix.

**Pull request merged**